crates/iota-single-node-benchmark/src/mock_storage.rs

```diff
@@ -71,8 +71,7 @@ impl InMemoryObjectStore {
                         .get_or_init(|| {
                             epoch_store
                                 .get_assigned_shared_object_versions(tx_key)
-                                .expect("get_assigned_shared_object_versions should not fail")
-                                .map(|l| l.into_iter().collect())
+                                .map(|versions| versions.into_iter().collect::<HashMap<_, _>>())
```

Comment:
```suggestion
                                .map(|versions| versions.into_iter().collect())
```
The target type can be inferred by the compiler here, so the turbofish is not needed.